### PR TITLE
Bump version and document release

### PR DESCRIPTION
**Pull request by jdtsmith**

### CODE REVIEW

**Comment by jdtsmith** [comment]
Hi @casouri: a ping on this PR, which documents the recent changes to navigation among all saved nodes, and increments the version.

File: vundo-diff.el [hunk 1]
```diff
@@ -132,7 +132,7 @@ NODE defaults to the current node."
 (defun vundo-diff--quit ()
   "Quit the `vundo-diff' window and possibly kill buffer."
   (let* ((buf (get-buffer (concat "*vundo-diff-" (buffer-name) "*")))
-	 (win (and buf (get-buffer-window buf)))
+	     (win (and buf (get-buffer-window buf)))
          (kill (eq vundo-diff-quit 'kill)))
```
Comment (casouri) [inline vundo-diff.el:136]: Is this intentional?

**Comment by casouri** [comment]
Thanks for the reminder, I have one comment, other than that it looks good.

File: vundo-diff.el (hunk 1 above)
Comment (jdtsmith) [reply vundo-diff.el:136]: Just the usual tab shenanigans I think, cleaned up.

**Comment by casouri** [comment]
Thanks, merged.

**Pull request closed without merging**